troposphere/static/js/components/MaintenanceScreen.react.js

```diff
@@ -32,7 +32,10 @@ export default React.createClass({
                     <img src={login} />
                 </div>
             </div>
-            <h4><h4><p> Atmosphere is currently under maintenance. </p> {statusPageEl}</h4></h4>
+            <h4 className="t-title">
```

Comment:
I'm guessing .esformatter was responsible for the weird output. It throws up when there is questionable html. Here it's probablyt he fact that there were nested `<h4>`. Have you run the formatter before? You might want to try it on this branch.